qa/tasks/swift.py

```diff
@@ -96,7 +96,7 @@ def create_users(ctx, config):
     try:
         yield
     finally:
-        for client in config['clients']:
+        for client in config.keys():
```

Comment:
this caused a regression in https://tracker.ceph.com/issues/44898

Reply:
hm, interesting, @shyukri was this change in original patch?

Reply:
Probably this patch has been missed ba3c17c4fad8672d95af0f3cb9e0fb4802d712e6

Reply:
please see https://github.com/ceph/ceph/pull/34379

Reply:
@cbodley seems fine, is this the only issue which has been found?

Reply:
not sure, i just saw the tracker issue and haven't run any tests against mimic